plan/2026-04-23-secret-in-log-cross-step-taint.md

```
@@ -0,0 +1,106 @@
+
+### Out of scope（follow-up として別 issue）
+- クロス job 伝播 (`needs.*.outputs.*`)（#432）
+- reusable workflow 跨ぎ (#433)
+- `${NAME}` / bash-specific な変数展開拡張のコーナーケース
+- `$GITHUB_ENV` に書く際の **add-mask** 補助 auto-fix（MVP では当面、sink 側 step の既存 auto-fix で対処）
+
+## 実装方針
+
+### データフロー
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)